### DISCUSSION TITLE
TV mode?

### DISCUSSION THREAD

**Discussion by rpapi** [InfinityLoop1308/PipePipe#501] [Ideas]
Hi, big fan of the project, trying to encourage my family to use it on TVs, but it is proving difficult as the interface is (1) very touch dependant and (2) optimized for vertical use. Any chance at some point tweaks could be made to improve navigation on non-touch devices and/or horizontal use? 

Much appreciated!

**Comment by InfinityLoop1308** [comment]
Yeah we are thinking about it and some features have been planned. I think a start point is #459 , you can continue the discussion there.
